export_modification.go

```diff
@@ -0,0 +1,72 @@
+package libsquash
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+func (e *export) InsertLayer(parent string) (*layer, error) {
+	id, err := newID()
+	if err != nil {
+		return nil, err
+	}
+
+	layerConfig := newLayerConfig(id, parent, "squashed w/ docker-squash")
+	layerConfig.ContainerConfig().Cmd = []string{"/bin/sh", "-c", fmt.Sprintf("#(squash) from %s", parent[:12])}
```

Comment:
This is the docker "Cmd" for the squash layer.  It serves two functions:

0. it is required as part of the config
0. we can use it to identify that a layer is our "squash" layer